content/docs/cucumber/cucumber-expressions.md

```diff
@@ -80,14 +80,17 @@ output parameters to your own types. Consider this Cucumber Expression:
 If we want the `{color}` output parameter to be converted to a `Color` object,
 we can define a custom parameter type in Cucumber's [configuration](/docs/cucumber/configuration).
 
+{{% block "java8" %}}
+```java
+ ParameterType("color", "red|blue|yellow", Color::new);
```

Comment:
from the previous docs, it would be good just to hint the usage of each param? So putting them on multiline with inline comments would help.